packages/csharp/PortingAssistant/PortingAssistant.Telemetry/Models/MetricsBase.cs

```diff
@@ -1,5 +1,10 @@
+using System;
```

Comment:
Future work log: This will conflict MetricsBase.cs codes. No need to merge this file. Required codes are already in MetricsBase.cs file.